src/wp-admin/includes/image.php

```diff
@@ -341,6 +341,18 @@ function wp_create_image_subsizes( $file, $attachment_id ) {
 				 */
 				if ( $scale_down ) {
 					$saved = $editor->save( $editor->generate_filename( 'scaled' ) );
+				} elseif ( $convert ) {
```

Comment:
It not fix the issue for WebP convert. See https://github.com/WordPress/wordpress-develop/pull/7733#issuecomment-2463921563

Reply:
Resolved with the regext fix.